Update bbfreeze

doadin opened the pull request:
with these changes its compatible with newer altgraph versions and uses and is compatible with new modulegraph version.

doadin committed:
update modulegraph

doadin committed:
[bbfreeze][getdeps]raise import error if pefile is not installed

doadin committed:
update setup.py required modules and travis/tox

pefile uses date and only has 1 available currently so requiring a
version number is probably why pefile doesnt get installed with bbfreeze
currently.

doadin committed:
bump version and update manifest

schmir commented:
Thanks for your pull request.

I'm not maintaining bbfreeze anymore and won't merge your changes. Sorry.

doadin committed:
update travis and ignore modulegraph in flake8/isort and run those

remove unsupported 3.2 and disable 3.x for now.
for simplicity of merging later versions ignore this in tests that way
we dont have to worry about it.

doadin committed:
simplify version

doadin committed:
Fix NamespacePackages

In testing zope was the only one not handled correctly.
I suspect this is due to lack of __init__.py and the if should be
changed however more testing is needed to verify

doadin committed:
Rename

doadin committed:
bump version

doadin committed:
remove typo

doadin committed:
fix init

doadin committed:
update setup with current info

doadin committed:
2to3

doadin committed:
Revert "2to3"

This reverts commit d6eb604f07ed5c15e343a6bd8bb234c66d6167ea.

doadin committed:
update readme and changelog

doadin committed:
Update prints
